src/services/mainnet-squid/query.ts

```diff
@@ -0,0 +1,35 @@
+import { createQuery, poolQuery } from '@/subsocial-query'
+import { gql } from 'graphql-request'
+import { GetProfilesQuery, GetProfilesQueryVariables } from './generated'
+import { mainnetSquidRequest } from './utils'
+
+const GET_PROFILES = gql`
+  query GetProfiles($addresses: [String!]!) {
+    accounts(where: { id_in: $addresses }) {
```

Comment:
maybe better id_eq, it should be faster

Reply:
but in this one, it will fetch multiple addresses, can't use `id_eq`